src/transcription/preRecordedTranscription.ts

```diff
@@ -44,15 +44,6 @@ export const preRecordedTranscription = async (
 ): Promise<PrerecordedTranscriptionResponse> => {
   const transcriptionOptions = { ...{}, ...options };
 
-  if (
-    !isUrlSource(source) &&
-    (source.mimetype === undefined || source.mimetype.length === 0)
-  ) {
-    throw new Error(
-      "DG: Mimetype must be provided if the source is a Buffer or a ReadStream"
-    );
-  }
-
```

Comment:
Do we need to remove the `Content-Type` header from line 61 below?

This is what we currently do:
```
const requestOptions: { [name: string]: { [key: string]: string } } = {};
  if (!isUrlSource(source)) {
    requestOptions.headers = {
      "Content-Type": source.mimetype,
    };
  }
  ```